test/utility_branded_token/decrease_supply.js

```diff
@@ -12,109 +12,115 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const utils = require('../test_lib/utils'),
-  UtilityBrandedTokenUtils = require('./utils'),
-  AccountProvider = utils.AccountProvider,
-  {Event} = require('../test_lib/event_decoder.js');
+const utils = require('../test_lib/utils');
+const UtilityBrandedTokenUtils = require('./utils');
+const { Event } = require('../test_lib/event_decoder.js');
 
 contract('UtilityBrandedToken::decreaseSupply', async (accounts) => {
```

Comment:
Why these test files and below files are being shown as new changes?
It seems the code is same with assert.strictEqual update.